pkg/operator/apis/monitoring/v1/types.go

```diff
@@ -242,6 +263,14 @@ type ClusterPodMonitoring struct {
 	Status PodMonitoringStatus `json:"status"`
 }
 
+func (p *ClusterPodMonitoring) GetKey() string {
+	return fmt.Sprintf("ClusterPodMonitoring/%s", p.Name)
+}
+
+func (p *ClusterPodMonitoring) GetStatus() *PodMonitoringStatus {
```

Comment:
Can you point me to where this is being used? Somewhere in #301?

Reply:
Correct.